packages/react-magma-dom/src/components/DatePicker/CalendarMonth.tsx

```diff
@@ -143,7 +143,12 @@ export const CalendarMonth: React.FunctionComponent<CalendarMonthProps> = (
   const startOfWeek = days.indexOf(i18n.datePicker.startOfWeek);
   const sortedDays = days.slice(startOfWeek).concat(days.slice(0, startOfWeek));
   const tableDaysHeaders = sortedDays.map((day, index) => (
-    <Th key={index} theme={theme} isInverse={context.isInverse}>
+    <Th
+      key={index}
+      theme={theme}
+      isInverse={context.isInverse}
+      aria-label={i18n.days.long[day]}
```

Comment:
Do we need to account for the locale?

For example, here we have an example that uses `locale: es.` Should the days match that language?
https://docs-preview-1528--upbeat-sinoussi-f675aa.netlify.app/api/date-pickers/#internationalization

Reply:
Upon further investigation, I think we just need to update that i18n example to include the following and it will work as expected
```
days: {
    long: {
      monday: 'Lunes',
      tuesday: 'Martes',
      wednesday: 'Miércoles',
      thursday: 'Jueves',
      friday: 'Viernes',
      saturday: 'Sábado',
      sunday: 'Domingo',
    },
```